test/text/glyph_manager.test.cpp

```diff
@@ -253,6 +253,57 @@ TEST(GlyphManager, LoadLocalCJKGlyph) {
         });
 }
 
+TEST(GlyphManager, LoadLocalCJKGlyphAfterLoadingRangeFromURL) {
```

Comment:
@pozdnyakov I've added a unit test that reproduces the issue. The fixture is pretty large - but I haven't got an idea of how to create a more synthetic fixture for this test.

It requests 々(0x3005), which is in the range 12244-12543. This range overlaps with Katakana letters which are expected to be locally generated. Without the changes in `GlyphManager`, this test will use the glyph fetched for the pbf for the katakana letter - and fails.